knock/src/core/ui/Reaction/Reaction.tsx

```diff
@@ -0,0 +1,55 @@
+import BeforeLike from '../../assets/icon/Before-like.svg';
+import BeforeHate from '../../assets/icon/Before-hate.svg';
+import AfterLike from '../../assets/icon/After-like.svg';
+import AfterHate from '../../assets/icon/After-hate.svg';
+import styles from '../../styles/Reaction.module.scss';
+import { useState } from 'react';
+
+interface ReactionProps {}
+
+const Reaction = ({}: ReactionProps) => {
+  const [likeCount, setLikeCount] = useState(0);
+  const [liked, setLiked] = useState(false);
+  const [unLiked, setUnLiked] = useState(false);
+
+  const handleLikeClick = () => {
+    const count = likeCount + 1;
+    setLikeCount(count);
+    setLiked(true);
+    setUnLiked(false);
+  };
+
+  const handleUnLikeClick = () => {
+    setUnLiked(!unLiked);
+    setLiked(false);
+    setLikeCount(0);
+  };
+
+  return (
+    <div className={`${styles['container']}`}>
+      <label className={`${styles['label']}`} onClick={handleLikeClick}>
+        <img src={liked ? AfterLike : BeforeLike} alt="좋아요" />
+        <button
+          className={`
+          ${styles['text']}
+          ${liked ? styles['like'] : ''}
+          `}
+        >
+          좋아요 {liked ? likeCount : ''}
+        </button>
+      </label>
```

Comment:
semantic tag 사용에 있어서 label은 input 태그 등에 이름을 붙여줄 때(for을 활용하여) 사용하고, 
지금은 범위 지정의 의미를 가지고 있으니 div가 더 적절하다고 생각됩니다 😄

[reference label태그 란?](https://www.tcpschool.com/html-tags/label)
```suggestion
      <div className={`${styles['label']}`} onClick={handleLikeClick}>
        <img src={liked ? AfterLike : BeforeLike} alt="좋아요" />
        <button
          className={`
          ${styles['text']}
          ${liked ? styles['like'] : ''}
          `}
        >
          좋아요 {liked ? likeCount : ''}
        </button>
      </div>
```